src/WebJobs.Script/Environment/EnvironmentExtensions.cs

```diff
@@ -284,6 +284,18 @@ public static bool IsAppService(this IEnvironment environment)
             return !string.IsNullOrEmpty(environment.GetEnvironmentVariable(AzureWebsiteInstanceId));
         }
 
+        /// <summary>
+        /// Gets a value indicating whether the application is running in Kuberneted App Service
+        /// </summary>
+        /// <param name="environment">The environment to verify</param>
+        /// <returns><see cref="true"/> If running in a Kubernetes Azure App Service; otherwise, false.</returns>
+        public static bool IsKubernetesAppService(this IEnvironment environment)
+        {
+            return !string.IsNullOrEmpty(environment.GetEnvironmentVariable(KubernetesServiceHost))
```

Comment:
IsAppService returns false for these apps I assume? If so, rather than naming this IsKubernetesAppService, I'd name it something else, since if you look at existing settings like IsLinuxAppService your name implies both. How about IsKubernetesManagedHosting, or IsKubernetesEnvironment or something?

Reply:
IsKubernetesManagedHosting - aligns better for the K8SE.